cmd/buildah/main.go

```diff
@@ -69,82 +69,16 @@ func main() {
 		return nil
 	}
 	app.Commands = []cli.Command{
-		{
-			Name:        "from",
-			Usage:       "Create a working container based on an image",
-			Description: fromDescription,
-			Flags:       fromFlags,
-			Action:      fromCmd,
-			ArgsUsage:   "IMAGE [CONTAINER-NAME]",
-		},
-		{
-			Name:        "list",
-			Usage:       "List working containers and their base images",
-			Description: listDescription,
-			Flags:       listFlags,
-			Action:      listCmd,
-			ArgsUsage:   " ",
-		},
-		{
-			Name:        "mount",
-			Usage:       "Mount a working container's filesystem root",
-			Description: mountDescription,
-			Action:      mountCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID",
-		},
-		{
-			Name:        "umount",
-			Aliases:     []string{"unmount"},
-			Usage:       "Unmount a working container's filesystem root",
-			Description: umountDescription,
-			Action:      umountCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID",
-		},
-		{
-			Name:        "add",
-			Usage:       "Add content to the container",
-			Description: addDescription,
-			Action:      addCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID [[FILE | DIRECTORY | URL] ...] [DESTINATION]",
-		},
-		{
-			Name:        "copy",
-			Usage:       "Copy content into the container",
-			Description: copyDescription,
-			Action:      copyCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID [[FILE | DIRECTORY | URL] ...] [DESTINATION]",
-		},
-		{
-			Name:        "run",
-			Usage:       "Run a command inside of the container",
-			Description: runDescription,
-			Flags:       runFlags,
-			Action:      runCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID COMMAND [ARGS [...]]",
-		},
-		{
-			Name:        "config",
-			Usage:       "Update image configuration settings",
-			Description: configDescription,
-			Flags:       configFlags,
-			Action:      configCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID",
-		},
-		{
-			Name:        "commit",
-			Usage:       "Create an image from a working container",
-			Description: commitDescription,
-			Flags:       commitFlags,
-			Action:      commitCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID IMAGE",
-		},
-		{
-			Name:        "delete",
-			Usage:       "Delete working container(s)",
-			Description: deleteDescription,
-			Action:      deleteCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID [...]",
-		},
+		addCommand,
```

Comment:
Do you need an import for these commands as they're in other source files?  I'm not seeing at least a '/cmd' in the imports and thought there'd be one (but may be showing my lack of go xp).